backend/fregepoc/repositories/tasks.py

```diff
@@ -60,6 +61,8 @@ def crawl_repos_task(indexer_class_name):
             args=(indexer_class_name,),
             countdown=indexer.rate_limit_timeout.seconds,
         )
+    else:
```

Comment:
IMO `for...else` structure is not a good practice. Perhaps, `if not batch` will be more readable?

Reply:
I didn't use `for...else`. It may look like that. However, this `else` is connected with `if` above.

Reply:
Oh, okay, sorry, my bad 😅